offer-manager-domain/src/main/groovy/life/qbic/business/products/create/CreateProductDataSource.groovy

```diff
@@ -13,10 +14,11 @@ interface CreateProductDataSource {
     /**
      * Stores a product in the database
      * @param product The product that needs to be stored
+     * @return The product identifier of the just created product
      * @since 1.0.0
      * @throws DatabaseQueryException if any technical interaction with the data source fails
      * @throws ProductExistsException if the product already exists in the data source
      */
-    void store(Product product) throws DatabaseQueryException, ProductExistsException
+    ProductId store(Product product) throws DatabaseQueryException, ProductExistsException
```

Comment:
Especially for interfaces, it is important to update the JavaDoc whenever changes occur. Thus I would kindly ask you to add the `@return` annotation with a nice description here. (whichever way you decide to go)

Reply:
I did update the return... you just made a suggestion for it in the comment above